rust/agama-cli/src/logs.rs

```diff
@@ -76,6 +92,24 @@ fn show(show: bool, text: &str) {
     print!("{}", text);
 }
 
+// Configurable parameters of the "agama logs" which can be
+// set by user when calling a (sub)command
+struct LogOptions {
+    paths: Vec<String>,
+    commands: Vec<String>,
+    verbose: bool,
+}
+
+impl LogOptions {
```

Comment:
```rust
impl Default for LogOptions {
    fn default() -> Self {
        Self {
            paths: DEFAULT_PATHS.iter().map(ToString::to_string).collect(),
            commands: DEFAULT_COMMANDS.iter().map(ToString::to_string).collect(),
            verbose: false,
        }
    }
}
```

Reply:
already implemented as a consequence of previous sugestions ;-)